src/libs/actions/IOU.js

```diff
@@ -119,10 +119,14 @@ function createIOUSplit(params) {
  * @param {Number} params.chatReportID
  * @param {String} params.transactionID
  * @param {String} params.comment
+ * @param {Object} params.iou
```

Comment:
This can be removed now

Reply:
Not sure how this got added 😂
I remember removing it.